### ISSUE TITLE
ZeroDivisionError while node syncing

### ISSUE THREAD

**Issue by MrHash** [prusnak/suez#24]
Same error as #16 received seemingly while LND 0.12.1 is in a syncing state.

**Comment by prusnak** [comment]
Can you please send a full traceback?

**Comment by MrHash** [comment]
```
Traceback (most recent call last):
  File "./suez", line 5, in <module>
    suez()
  File "/root/.cache/pypoetry/virtualenvs/suez--l8OAkrR-py3.7/lib/python3.7/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/root/.cache/pypoetry/virtualenvs/suez--l8OAkrR-py3.7/lib/python3.7/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/root/.cache/pypoetry/virtualenvs/suez--l8OAkrR-py3.7/lib/python3.7/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/root/.cache/pypoetry/virtualenvs/suez--l8OAkrR-py3.7/lib/python3.7/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/root/downloads/suez/suez.py", line 211, in suez
    table = channel_table(ln, score, show_remote_fees, show_chan_ids)
  File "/root/downloads/suez/suez.py", line 75, in channel_table
    uptime = 100 * c.uptime // c.lifetime
ZeroDivisionError: integer division or modulo by zero
```

**Comment by prusnak** [comment]
Fixed in d943cc1094b2108ad847f75f2fa2682e17a2f817

Thanks!
